source/wp-content/themes/wporg-developer-2023/js/code-tabs.js

```diff
@@ -0,0 +1,51 @@
+const init = () => {
+	const tabs = document.querySelectorAll( '.code-tabs' );
+
+	if ( ! tabs.length ) {
+		return;
+	}
+
+	tabs.forEach( ( tab ) => {
+		const buttons = tab.querySelectorAll( 'button' );
+
+		buttons.forEach( ( button ) => {
+			button.classList.add( 'wp-block-button__link' );
+
+			if ( button.classList.contains( 'is-active' ) ) {
+				button.setAttribute( 'aria-pressed', 'true' );
+			}
+		} );
+
+		// Block button styles require a parent with `class*="wp-block"`.
+		tab.classList.add( 'wp-block', 'is-small', 'is-style-toggle' );
+
+		tab.addEventListener( 'click', ( event ) => {
+			event.preventDefault();
+
+			if ( event.target.classList.contains( 'wp-block-button__link' ) ) {
```

Comment:
Does this have to check for a Gutenberg-provided class?

Reply:
👍  Updated this now to use the existing class, thanks